### PR TITLE
Feature/logout url variable

### PR DESCRIPTION
**Pull request by bobbi-hbrown**
https://dluhcdigital.atlassian.net/jira/software/c/projects/SMD/boards/140?quickFilter=815&selectedIssue=SMD-346

Allows for an override of the g.logout_url variable constructed by the _build_logout_url function. If the config_var_logout_url_override variable is present in a project's config, any unauthenticated requests will be redirected to the URL supplied here instead of using Authenticator's default login screen.

### CODE REVIEW

**Comment by cyrusdobbs** [comment]
Looks like your latest commit removed your change to `pyproject.toml`

**Comment by bobbi-hbrown** [comment]
> Looks like your latest commit removed your change to `pyproject.toml`

I removed this because there was a Github action which incremented it automatically, and was failing with my manual version change.. but this time doesn't appear to have run. Will look into this further

**Comment by cyrusdobbs** [comment]
> > Looks like your latest commit removed your change to `pyproject.toml`
> 
> I removed this because there was a Github action which incremented it automatically, and was failing with my manual version change.. but this time doesn't appear to have run. Will look into this further


Ah yeah - the README states it will just be done via a Github action. This is new since I last pushed to this repo. https://github.com/communitiesuk/funding-service-design-utils#releasing

**Comment by bobbi-hbrown** [comment]
> > > Looks like your latest commit removed your change to `pyproject.toml`
> > 
> > 
> > I removed this because there was a Github action which incremented it automatically, and was failing with my manual version change.. but this time doesn't appear to have run. Will look into this further
> 
> Ah yeah - the README states it will just be done via a Github action. This is new since I last pushed to this repo. https://github.com/communitiesuk/funding-service-design-utils#releasing

I think this will eventially run on merge!

File: fsd_utils/authentication/config.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import enum
 
 config_var_auth_host = "AUTHENTICATOR_HOST"
+config_var_logout_url_override = "LOGOUT_URL_OVERRIDE"
```
Comment (tferns) [inline fsd_utils/authentication/config.py:12]: Would prefer all capitals for constants, but understand you've just kept the existing convention so no need to worry about it

**Review by tferns — APPROVED** [review]

**Pull request merged**